docs/requirements.txt

```diff
@@ -0,0 +1,10 @@
+avro
```

Comment:
Agreed, just noticed there's an option for RTD to install the project, removing this dependency now